keptn-jenkins-library/5.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,18 @@
+version: '5.0'
+name: keptn-jenkins-library
```

Comment:
Would use `jenkins-service` to be consistent with other services.

Reply:
It's actually not a service, it's a shared library to include in your Jenkins Configuration rather than direclty in Keptn.